deepLuna.py

```diff
@@ -401,158 +401,122 @@ def correct_day_subtable(subTable):
+            if file_scene in scr_table.scene_names():
+                # Ingest this file and mutate our translation table
                 file_path = os.path.join(root, filename)
-                newTable = new_txtfile_update_table(file_path, scrTable, newTable, dataScriptJP)
-                os.remove(file_path)
-                print(file_path+" done")
-
-
-def txtfile_update_table(dayFile,table_scr,mainTable,scriptTextMrgStream):
-
-    cleanTextFile = open(dayFile, "r+", encoding="utf-8")
-    cleanText = cleanTextFile.read()
-
-    cleanText = re.sub(r"\n\<Page[0-9]+\>\n",r"\n",cleanText)
-    cleanText = re.sub(r"\<Page[0-9]+\>\n",r"",cleanText)
-    cleanText = re.sub(r"C\:\>",r"",cleanText)
-    cleanText = re.sub(r"C\:\>",r"",cleanText)
-    cleanText = re.split(r"\n|\#",cleanText)
-
-    redName = os.path.basename(dayFile).split('.')[0]
-
-    for day in table_scr:
-        if day[0] == redName:
-            assoDay = day[1]
-            break
-
-    dayTable = []
-    counter = 0
-
-    newMainTable = mainTable
-
-    for i in range(len(assoDay)):
-
-        pageUpdated = page_to_SpeLineInfo(assoDay[i])
-
-        for l in range(len(pageUpdated)):
-            extrLine = extract_by_line(scriptTextMrgStream,int(pageUpdated[l][0])+1,i+1,redName,pageUpdated[l][1],pageUpdated[l][2])
-            extrLine[2] = extrLine[2] if cleanText[counter] == extrLine[1] else cleanText[counter]
-            dayTable.append(extrLine)
-            counter += 1
-
-    for i in  range(len(dayTable)):
-        for j in range(len(newMainTable)):
-            if dayTable[i][0] == newMainTable[j][0] and dayTable[i][2] != 'TRANSLATION':
-                newMainTable[j][2] = dayTable[i][2]
-
-    return(newMainTable)
-
-    cleanTextFile.close()
+                translation_table = new_txtfile_update_table(
+                    file_path, scr_table, translation_table, jp_script_mrg)
 
+                # Remove after processing
+                os.remove(file_path)
+                print("Imported changes from '%s'" % file_path)
 
-def new_txtfile_update_table(dayFile,table_scr,mainTable,scriptTextMrgStream):
 
+def new_txtfile_update_table(filename, scr_table, translation_table, jp_script_mrg):
     # Open scene file
-    with open(dayFile, "r+", encoding="utf-8") as cleanTextFile:
+    with open(filename, "r+", encoding="utf-8") as cleanTextFile:
         cleanText = cleanTextFile.read()
 
-    # We eliminate choices indication and split the text into pages (and since we start by page 1, we eliminate the first element of the list)
-    cleanText = re.sub(r"C\:\>",r"",cleanText)
-    cleanText = re.sub(r"C\:\>",r"",cleanText)
-    cleanText = re.split(r"\<Page[0-9 ]+\>",cleanText)[1:]
+    # We eliminate choices indication and split the text into pages
+    # And since we start by page 1, we eliminate the first element of the list
+    cleanText = re.sub(r"C\:\>", r"", cleanText)
+    cleanText = re.sub(r"C\:\>", r"", cleanText)  # TODO(ross): dupe?

```

Comment:
set a global flag on the regex instead of running it twice? Or is this expecting there to be some sort of nested C:\ like `C:C:\\`?

Reply:
I believe this is non-intentional, sorry. Theoretically the substitution _should_ work when it's already done once, so yeah, we can just erase it.